.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,86 @@
+## Type of change
+
+<!-- Mandatory
+Please delete options that are not relevant
+-->
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Enhancement
+- [ ] Breaking change
+
+## What does this PR do?
+
+<!-- Mandatory
+Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
+-->
+
+## Why is it important?
+
+<!-- Mandatory
+Explain here the WHY, or the rationale/motivation for the changes.
+-->
+
+## Checklist
+
+<!-- Mandatory
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply.
+-->
+
+- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/). 
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing **unit** tests pass locally with my changes
+- [ ] New and existing **integration** tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
It wouldn't work without them right?

Reply:
Resolving as this item was removed, thanks!

Reply:
Well, depending on how dependencies are handled, you could be using some development version. Nit, I would say _upstream_